patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -32,17 +34,30 @@ This pattern applies very widely from InnerSource initiatives in their infancy s
 * Some organizations in some countries may face extra complexity when introducing metrics as the countries may not allow tracking individuals.
 * There might be a learning curve in the discussion about metrics. And perhaps the tools do not support the InnerSource metrics we are looking for.
 
+In favour:
+
+* Management needs to understand how InnerSource is impacting development. Metrics allow for accurate visualizations.
+* There are several industry standards for monitoring the software development process (DORA, CHAOSS, ...).
+* There are many tools providing metrics for monitoring the software development process and the software product.
+
 ## Solution
 
+### Manage people's needs
+
 Bring developers, middle managers and C-level to have a discussion about metrics. And consider other roles out of the usual development process such as Human Resources, legal departments, product management, and others.
 
 Let developers and middle managers know that these metrics or KPIs are not focused on tracking their personal performance, but to compare if the initiative is currently working as expected.
 
 Consider a third party that is seen as a neutral player to produce such metrics.
 
+### Approach professionaly
+
 Have specific training on the topic of metrics and good practices to use them. An example is to have a methodology to follow metrics such as the Goal-Question-Metric approach or the Objectives-KeyResults one. On the other hand, try to reflect the short-term and medium-term goals in the metrics to be used.
 
-Metrics when published or discussed should be done so in the aggregate without referring to specific people.
+Metrics when published or discussed should
+
+* be aggregated, without referring to specific people,
+* watching out for outliers.
 
 Produce a characterization of metrics as this might be helpful for others to understand and follow.
```

Comment:
Do you have an example of what is meant by "characterization"?

Reply:
It's an old text. [It was there before the move in jul 2020](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/aecdd1821b0f091e1a88c8318df52c58fdfa789b/patterns/2-structured/introducing-metrics-in-innersource.md).

Reply:
Oh yeah, didn't see the bullet's right behind it. Thanks!